basenames.c

```diff
@@ -5,44 +5,44 @@
 
 /* Return NAME with any leading path stripped off.  */
 
-char *ourBaseName(char *name)
-{
-    char *base;
+char *ourBaseName(char *name) {
+	char *base;
 
-    base = strrchr(name, '/');
-    return base ? base + 1 : name;
+	base = strrchr(name, '/');
+	return base ? base + 1 : name;
 }
 
-static void stripTrailingSlashes(char *path)
-{
-    char *last;
+static void stripTrailingSlashes(char *path) {
+	char *last;
 
-    last = path + strlen(path) - 1;
-    while (last > path && *last == '/')
+	last = path + strlen(path) - 1;
+	while (last > path && *last == '/') {
 	*last-- = '\0';
+	}
 }
 
-char *ourDirName(char *origname)
-{
-    char *slash;
-    char *name;
+char *ourDirName(char *origname) {
+	char *slash;
+	char *name;
 
-    name = strdup(origname);
+	name = strdup(origname);
 
-    stripTrailingSlashes(name);
+	stripTrailingSlashes(name);
 
-    slash = strrchr(name, '/');
+	slash = strrchr(name, '/');
 
-    if (!slash) {
+	if (!slash) {
 	/* No slash, must be current directory */
 	free(name);
 	/* strdup used, as the return value will be free()ed at some point */
 	return strdup(".");
-    } else {
+	}
+	else {
```

Comment:
Why? Because I am human. I just fixed the indention in over 3,000 lines of code. If I missed some sorry. But thank you for pointing out one that I missed.